common/config/azure-pipelines/jobs/docs/docs-build.yaml

```diff
@@ -0,0 +1,94 @@
+# The template defines the primary steps for creating the TypeDoc API files, gathering other md files,
+# and running a step to combine everything.
+
+parameters:
+  - name: workingDir
+    type: string
+    default: $(Pipeline.Workspace)/itwinjs-core
+  - name: stagingDir
+    type: string
+    default: $(System.ArtifactsDirectory)/staging
+  - name: outputDir
+    type: string
+    default: $(Pipeline.Workspace)/tempDocsBuild
+  - name: downloadCurrentBuildArtifacts
+    type: boolean
+    default: false
+  - name: shouldPublish
+    type: boolean
+    default: false
+
+jobs:
+  - job:
+    workspace:
+      clean: all
+    pool:
+      name: iModelTechCI
+      demands: Agent.OS -equals Windows_NT
+
+    steps:
+      - checkout: self
+        path: itwinjs-core
+        clean: true
+
+      - task: NodeTool@0
+        displayName: Use Node 14
+        inputs:
+          versionSpec: 14.17.4
```

Comment:
Can we update this to Node 18 or is there some restriction?

Reply:
I tried bumping up to node 16 everywhere in the docs pipeline, but that started causing test failures so I just reverted back to node 14 everywhere. I think I could probably update the node version everywhere except the docs product tests without issue, but not sure if we want to keep our node version consistent throughout or not.

Reply:
Okay, let's leave it as is for now and then follow up in a separate PR.